docs/tutorials/cert-manager-and-trust-manager.md

```diff
@@ -0,0 +1,274 @@
+Example of the broker CRD is located in the local repository cloned during activities from section **Deploy ArtemisCloud operator**.
+
+Replace **myproject** with actual namespace your brokers are installed in.
+
+Deploy a broker with (beeing on the root folder of this repository):
+```shell
+kubectl apply -f docs/examples/artemis/artemis_ssl_acceptor_cert_and_trust_managers.yaml -n myproject
+```
+
+### Deploy a queue
+Example of the queue CRD is located in the local repository cloned during activities from section **Deploy ArtemisCloud operator**.
+
+Deploy a queue with (beeing on the root folder of this repository):
+```shell
+kubectl apply -f docs/examples/address/address_queue.yaml -n myproject
+```
+
+The CR tells the Operator to create a queue named myQueue0 on address myAddress0 on each broker that it manages.
+
+After the CR is deployed, you can observe the queue on the broker:
+```shell
+kubectl exec artemis-broker-ss-0 -n myproject -- /bin/bash /home/jboss/amq-broker/bin/artemis queue stat --user admin --password admin --url tcp://artemis-broker-ss-0:61616
+OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
+Connection brokerURL = tcp://artemis-broker-ss-0:61616
+|NAME                     |ADDRESS                  |CONSUMER_COUNT |MESSAGE_COUNT |MESSAGES_ADDED |DELIVERING_COUNT |MESSAGES_ACKED |SCHEDULED_COUNT |ROUTING_TYPE |
+|DLQ                      |DLQ                      |0              |0             |0              |0                |0              |0               |ANYCAST      |
+|ExpiryQueue              |ExpiryQueue              |0              |0             |0              |0                |0              |0               |ANYCAST      |
+|myQueue0                 |myAddress0               |0              |0             |0              |0                |0              |0               |ANYCAST      |
+```
+
+### Test messaging over a SSL connection
+Log into the broker pod first to get a shell command environment:
+```shell
+kubectl exec --stdin --tty artemis-broker-ss-0 -n myproject -- /bin/bash
+[jboss@artemis-broker-ss-0 ~]$
+```
+
+Then send 100 messages through port 61618, but before replace **dummy_password** with the password you have used in section **Create secret for keystore password**.
+```shell
+cd amq-broker/bin
+[jboss@artemis-broker-ss-0 bin]$ ./artemis producer --user admin --password admin --url tcp://artemis-broker-ss-0:61618?sslEnabled=true\&keyStorePath=/amq/extra/secrets/amq-ssl-secret/keystore.jks\&keyStorePassword=dummy_password\&trustStorePath=/amq/extra/configmaps/ca-bundle/truststore.jks\&trustStorePassword=changeit --message-count 100
+OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
```

Comment:
when I run this command I got error
```
javax.net.ssl.SSLHandshakeException: No subject alternative DNS name matching artemis-broker-ss-0 found.
```
Is this because I didn't add artemis-broker-ss-0 to the cert?

Reply:
Yes, corrected now certificate includes artemis-broker-ss-0 dns.